### DISCUSSION TITLE
Star operator not working as intended?

### DISCUSSION THREAD

**Discussion by vekexasia** [JetBrains/ideavim#888] [General]
if you try to * over { character it does not allow u to navigate to all brackets. 

As far as I know this should be possible

**Comment by citizenmatt** [comment]
Looks like a bug! Star should search for the "next word", with various rules for finding the next word, but IdeaVim doesn't know what to do if it doesn't find it. Vim will search for the text under the caret, even if it's not a word, so `myFunc() {` would search for `{` while IdeaVim just does nothing. But given `myFunc() { return foo }`, both Vim and IdeaVim will search for `return`, even when the cursor is on `{`.

Here's the issue: [VIM-3445](https://youtrack.jetbrains.com/issue/VIM-3445/Star-operator-does-not-find-non-word-symbol-under-cursor)
